arrows/ffmpeg/ffmpeg_video_output.cxx

```diff
@@ -776,6 +778,17 @@ ::write_remaining_packets()
   while( write_next_packet() ) {}
 }
 
+// ----------------------------------------------------------------------------
+int64_t
+ffmpeg_video_output::impl::open_video_state
+::next_video_pts() const
+{
+  return static_cast< int64_t >(
+    static_cast< double >( frame_count ) *
+    video_stream->time_base.den / video_stream->time_base.num /
+    codec_context->framerate.num * codec_context->framerate.den + 0.5 );
```

Comment:
Not necessary, but it may be worth breaking this up or adding parentheses for readability